docs/physics/montecarlo/propagation.rst

```diff
@@ -150,103 +168,101 @@ Thomson scattering according to
-re-emission process occurs isotropically as well, the same directional sampling
-as described above is used. Energy conservation again dictates how the packet
-energy after the line interaction event is determined. The important difference
-is the assignment of the post-interaction frequency. This depends on the
-selected line interaction mode (see :doc:`Line Interaction Modes
-<lineinteraction>`).
+.. plot:: physics/pyplot/plot_mu_in_out_packet.py
 
 .. note::
 
     Note that the inclusion of special relativistic effects in TARDIS is at
-    best to first order in :math:`\beta`. 
+    best to first order in :math:`\beta`.
+
+Distance to Next Event
+----------------------
+
+With these assumptions, the accumulation of optical depth along a packet's trajectory currently proceeds according
+to the following scheme (which was originally introduced by :cite:`Mazzali1993`): 
+given the current lab-frame frequency of the packet, the distance to the next
+Sobolev point (i.e. to the next line resonance) is calculated as discussed above. Until this location, the packet continuously accumulates optical depth due to
+electron-scattering. At the Sobolev point, the accumulated optical depth is
+instantaneously incremented by the Sobolev optical depth. Afterwards, the
+procedure is repeated, now with respect to the next transition in the
+frequency-ordered list of all possible atomic line transitions. The point at
+which the accumulated optical depth surpasses the (randomly calculated) interaction optical depth determines the type of interaction the packet performs
+and at which location in the spatial mesh. The entire process is summarized in the sketch below (taken from :cite:`Noebauer2014`, adapted from
+:cite:`Mazzali1993`):
+
+.. image::
+    images/optical_depth_summation.png
+    :width: 500
+
+Three possible cases are highlighted in the diagram above, with the dotted lines showing the (randomly assigned) optical depth at which the packet interacts. In case I, the interaction optical
+depth value is reached on one of the path segments between successive Sobolev
+points, where the packet is accumulating electron scattering optical depth.
+Thus, the packet performs a Thomson scattering. In case II, the interaction
+optical depth is reached during the instantaneous increment by the line optical
+depth at one of the Sobolev points. As a consequence, the packet performs an
+interaction with the corresponding atomic line transition. In both of these cases, the packet is moved to the interaction location; a new propagation direction, energy, and frequency are assigned (as discussed above); and the process of accumulating optical depth starts over. Finally, if the packet reaches the shell boundary before the optical depth value necessary for a physical interaction is achieved (as in case III), the packet will be moved to the next cell, the plasma properties will be updated, and the accumulation of optical depth will continue in the next cell.
+
```

Comment:
This appears to be repetition of the parts you have added. Try adding the references and image from this section into your work